tests/Makefile

```diff
@@ -236,17 +236,17 @@ test-lz4-basic: lz4 datagen unlz4 lz4cat
 	./datagen -g256MB | $(LZ4) -vqB4D | $(LZ4) -t
 	@echo "hello world" > tmp
 	$(LZ4) --rm -f tmp
-	ls -ls tmp         && false || true   # must fail (--rm)
-	ls -ls tmp.lz4
-	$(PRGDIR)/lz4cat tmp.lz4              # must display hello world
-	ls -ls tmp.lz4
+	test ! -f tmp                      # must fail (--rm)
```

Comment:
`test` is POSIX -- http://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html -- so approximately as portable as other UNIXisms the Makefile depends on.

I don't know a lot about building UNIXy stuff under Windows, but lz4's Makefiles assume the presence of enough UNIX stuff that I'm pretty sure it's safe to assume you have `test` as well.